pkg/operator/management/management_state.go

```diff
@@ -0,0 +1,69 @@
+package management
+
+import (
+	"github.com/openshift/api/operator/v1"
+)
+
+var (
+	allowOperatorUnmanagedState = true
```

Comment:
why this global state?

Reply:
debated via slack, basically we can plumb this trough each individual controller as a bool field in struct, which won't make it less ugly and will require everybody that use library-go to update their controllers once they bump.